nih-sparc/sparc-app
# Event detail page cu-qn7eku

# Description

The purpose of this PR is to tweak the following for the event detail page:
- Change the main copy to be on the right side
- Add External Link section
- Add "copy permalink" button
- Add "View all Events" (or "View all News" on news page)

Note: The "View all" links do not work at the moment as they are linking to pages that don't yet exist.

## Type of change


# How Has This Been Tested?

- Go to an [event](http://localhost:3000/news-and-events/events/5i2AUXJ0JU57vDgaVpCqcz)
- Main content should be  on the right
- External link should be shown and clickable
- Clicking on the link icon under "Share" should copy the page's URL to the clipboard
- "View all events" at the bottom should be present
- Go to a [news page](http://localhost:3000/news-and-events/news/ffIZs24VWLGNftfw2uSyp)
- Main content should be on the left (when available, it's not in this example)
- The bottom back link should be "View All News"

# Checklist:
